app/src/main/java/eu/kanade/tachiyomi/source/LocalSource.kt

```diff
@@ -266,6 +266,16 @@ class LocalSource(
             .joinToString(", ") { it.trim() }
             .takeIf { it.isNotEmpty() }
             ?.let { manga.artist = it }
+
+        when (comicInfo.publishingStatusTachiyomi?.value) {
+            "Ongoing" -> manga.status = 1
+            "Completed" -> manga.status = 2
+            "Licensed" -> manga.status = 3
+            "Publishing finished" -> manga.status = 4
+            "Cancelled" -> manga.status = 5
+            "On hiatus" -> manga.status = 6
+            else -> { manga.status = 0 }
+        }
```

Comment:
Same here use the constant value in SManga

Also you can move out the assignment

```suggestion
        manga.status = when (comicInfo.publishingStatusTachiyomi?.value) {
            "Ongoing" ->  1
            "Completed" -> 2
            "Licensed" -> 3
            "Publishing finished" -> 4
            "Cancelled" -> 5
            "On hiatus" -> 6
            else -> 0
        }
```